### PR TITLE
Feature/sbachmei/auto update readme python version

### PR DESCRIPTION
**Pull request by stevebachmeier**
### Description

- *Category*:  documentation
- *JIRA issue*: na
- *Research reference*:  na

### Changes and notes
 

We should auto-update the allowable python versions in the README.

### Verification and Testing

Tested w/ fresh install.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -10,6 +10,8 @@ Research repository for the vivarium_nih_us_cvd project.
 Installation
 ------------
 
+Installation requires a Python version between 3.9 and 3.11.
```
Comment (stevebachmeier) [inline README.rst:13]: This got updated from "between foo and bar"  in my test

File: README.rst [hunk 2]
```diff
@@ -23,7 +25,7 @@ Once you have all three installed, you should open up your normal shell
 You'll then make an environment, clone this repository, then install
 all necessary requirements as follows::
 
-  :~$ conda create --name=vivarium_nih_us_cvd python=3.8
+  :~$ conda create --name=vivarium_nih_us_cvd python=<3.9-3.11>
```
Comment (stevebachmeier) [inline README.rst:28]: This got updated from "python=\<some versions\>" in my test

**Comment by mattkappel** [comment]
Why would you run setup before reading the readme?

File: setup.py [hunk 3]
```diff
@@ -17,7 +17,18 @@
...
+    readme = f.readlines()
+for i, line in enumerate(readme):
+    instruction_pattern = "Installation requires a Python version between"
+    if instruction_pattern in line:
+        readme[i] = f"{instruction_pattern} {min_version[1]} and {max_version[1]}.\n"
+    code_pattern = "  :~$ conda create --name=vivarium_nih_us_cvd python="
+    if code_pattern in line:
+        readme[i] = f"{code_pattern}<{min_version[1]}-{max_version[1]}>\n"
+with open("README.rst", "w") as f:
+    f.writelines(readme)
```
Comment (rmudambi) [inline setup.py:31]: This doesn't update the README that exists on GitHub, does it? I think that's most likely what people are reading.

Comment (stevebachmeier) [reply setup.py:31]: That's a good point, I wasn't thinking about this correctly.

**Pull request closed without merging**